checkpoint2/index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <title>Ryan's Checkpoint 2</title>
+    <link rel="stylesheet" href="/css/style.css">
+  </head>
+  <body>
+      <div class="background-image">
+    <header>
+      <h1>Ryan Roberts</h1>
+    </header>
+    <nav>
+        <a href="#about">About</a> &emsp;
+        <a href="#jobs">Professional Experience</a> &emsp;
+        <a href="#skills">Skills</a> &emsp;
+        <a href="#school">Education</a> &emsp;
+        <a href="#contact">Contact</a>
+    </nav>
+    <main>
+      <div class="photo">
+        <img src="ryan.jpg">
+      </div>
+      <div class="flexbox-container1">
+      <div class="flexbox-container2">
+        <div class="about" id="about">
```

Comment:
you have class='about' and id='about'... in the css you are only using the IDs.  having the class in there won't hurt the page display, but it is generally a bad practice to have class and ID names that are the same.  it's just confusing.